vpdq/pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[build-system]
+requires = [
+    "setuptools >= 61.0",
+    "wheel",
+    "cython",
+]
+build-backend = "setuptools.build_meta"
+
+[project]
+name="vpdq"
+authors=[{name="Facebook", email="threatexchange@fb.com"}]
```

Comment:
The new address is `threatexchange@meta.com`

Reply:
Should author name be updated to `Meta` as well?

Reply:
I don't think I've gotten any guidance on it, I think both are currently accepted.